### ISSUE TITLE
Add NextJs Site to data-portal for atlas pages

### ISSUE THREAD

**Issue by NoopDog** [clevercanary/hca-data-portal#17]
#### Need
We will be incrementally moving the hca data portal site from Gatsby to nextjs. We need a nextjs app in the same repository to start committing to.

#### Approach
Create a nextjs app under a `/next` directory from the root. This can be to get started the most basic next app with the default home page.  We will add customizations in a separate ticket and for now we just need the basic app checked in (client side only, no server for this app).

#### Notes
Once this is done PR to https://github.com/DataBiosphere/data-portal

**Comment by NoopDog** [comment]
complete
